modules/statistics/statistics.js

```diff
@@ -199,10 +203,13 @@ export default function Statistics(xmpp, options) {
 
     Statistics.instances.add(this);
 
+    conference && RTCStats.start(this.conference);
```

Comment:
There is no way conference is undefined, is there?

Reply:
I didn't know the full intent of the Statistics class (just group some logic that's tightly coupled to the JitsiConference class or more generic), technically it has functionality that can work without a conference. But as it's currently used I suppose conference will be there. I'll remove it.